.github/workflows/sync-flow-go.yml

```
@@ -32,11 +33,12 @@ jobs:
           repository: onflow/flow-go
       
       # get the latest update branch name to base the PR on that branch
+      # else, use the "feature-secure-cadence" branch
       - name: Get onflow/flow-go base branch name
         run: |
           git fetch
           BRANCH=$(git branch -r -l "*auto-cadence-upgrade/*" --format "%(refname:lstrip=3)" | sort -r | head -n 1)
-          [ -z $BRANCH ] && BRANCH=master
+          [ -z $BRANCH ] && BRANCH=feature-secure-cadence
```

Shall we name this branch `feature/secure-cadence`? In `flow-go` repo, branches starts with `feature/***` has some special semantics (e.g: Need reviews for merging, CI is enabled, etc.)
```suggestion
          [ -z $BRANCH ] && BRANCH=feature/secure-cadence
```